vars/odsPipeline.groovy

```diff
@@ -1,17 +1,5 @@
-import org.ods.OdsLogger
-import org.ods.OdsPipeline
-
 def call(Map config, Closure body) {
-
-  def debug = env.DEBUG
-  if (debug != null && config.debug == null) {
-    config.debug = debug
-  }
-
-  def logger = new OdsLogger(this, debug)
-  OdsPipeline pipeline = new OdsPipeline(this, logger)
-
-  pipeline.execute(config, body)
+  odsAppPipeline(config, body)
```

Comment:
Did you add this odsAppPipeline instead of doing whats in there here because you want to retire odsPipeline at some point or why this additional vars file?

Reply:
Yes correct. I will add an echo statement that mentions this.